templates/config.edn

```diff
@@ -1,137 +1,170 @@
 {:meta/version 1
 
- ;; Currently, we support either "Markdown" or "Org".
- ;; This can overwrite your global preference so that
- ;; maybe your personal preferred format is Org but you'd
- ;; need to use Markdown for some projects.
- ;; :preferred-format ""
-
- ;; Preferred workflow style.
- ;; Value is either ":now" for NOW/LATER style,
- ;; or ":todo" for TODO/DOING style.
+ ;; Set the preferred format.
+ ;; Available options:
+ ;; - "Markdown" (default)
+ ;; - "Org"
+ ;; Default value: "Markdown"
```

Comment:
If the option list is limited and you have pointed out the default value, maybe you don't need this line anymore?